docs/configuration/configuration.md

```diff
@@ -896,6 +896,7 @@ create a target group for every app that has at least one healthy task.
 The following meta labels are available on targets during [relabeling](#relabel_config):
 
 * `__meta_marathon_app`: the name of the app (with slashes replaced by dashes)
+* `__meta_marathon_container_port`: the docker container port
```

Comment:
Docker is in the following line.

Is is a Docker or any container management software?